apps/src/bin/anoma-client/cli.rs

```diff
@@ -97,6 +64,32 @@ fn craft_intent(
     );
     let data_bytes = signed_ft.try_to_vec().unwrap();
 
-    let mut file = File::create(file_path).unwrap();
-    file.write_all(&data_bytes).unwrap();
+    if to_stdout {
+        let mut out = std::io::stdout();
+        out.write_all(&data_bytes).unwrap();
+        out.flush().unwrap();
```

Comment:
I think this should be equivalent
```suggestion
        println!("{:?}", data_bytes);
```

Reply:
this is not equivalent